tests/main/cgroup-devices/test-snapd-service/bin/service

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+rm -f "$SNAP_COMMON/remove-me"
```

Comment:
Why this snap is defined as part of the test instead of adding that one to tests/lib/snaps ?

Reply:
Because that doesn't scale. This snap has single purpose. It is not meant to be reused and modified and extended. I looked at the "simple" service snaps and they were anything but. I just chose not to share.